src/datasources/postgres/ProposalBookingDataSource.ts

```diff
@@ -101,8 +101,11 @@ export default class PostgresProposalBookingDataSource
           : ProposalBookingStatus.DRAFT
       )
       .where('proposal_booking_id', id)
-      .andWhere('status', ProposalBookingStatus.ACTIVE)
-      .orWhere('status', ProposalBookingStatus.DRAFT)
+      .andWhere((query) =>
+        query
+          .where('status', ProposalBookingStatus.ACTIVE)
```

Comment:
Instead of `where` and `orWhere` You could use here `whereIn` https://knexjs.org/#Builder-whereIn
something like
```
.update(
        'status',
        action === ProposalBookingFinalizeAction.COMPLETE
          ? ProposalBookingStatus.COMPLETED
          : ProposalBookingStatus.DRAFT
      )
      .where('proposal_booking_id', id)
      .whereIn('status', [ProposalBookingStatus.ACTIVE, ProposalBookingStatus.DRAFT])
      .returning<ProposalBookingRecord[]>(['*']);
```